### PR TITLE
Add maintenance job doc

### PR DESCRIPTION
**Pull request by qiuming-best**
Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue)
Add maintenance job doc
Add parallel restore doc
# Please indicate you've done the following:

- [ ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: site/content/docs/main/customize-installation.md [hunk 1]
```diff
@@ -125,7 +128,11 @@ velero install \
   [--node-agent-pod-cpu-request <CPU_REQUEST>] \
   [--node-agent-pod-mem-request <MEMORY_REQUEST>] \
   [--node-agent-pod-cpu-limit <CPU_LIMIT>] \
-  [--node-agent-pod-mem-limit <MEMORY_LIMIT>]
+  [--node-agent-pod-mem-limit <MEMORY_LIMIT>] \
+  [--maintenance-job-cpu-request <CPU_REQUEST>] \
+  [--maintenance-job-mem-requestt <MEMORY_REQUEST>] \
```
Comment (ywk253100) [inline site/content/docs/main/customize-installation.md:133]: Typo for `request`

File: site/content/docs/main/velero-install.md [hunk 2]
```diff
@@ -26,7 +26,11 @@ velero install \
     [--node-agent-pod-cpu-request <CPU_REQUEST>] \
     [--node-agent-pod-mem-request <MEMORY_REQUEST>] \
     [--node-agent-pod-cpu-limit <CPU_LIMIT>] \
-    [--node-agent-pod-mem-limit <MEMORY_LIMIT>]
+    [--node-agent-pod-mem-limit <MEMORY_LIMIT>] \
+    [--maintenance-job-cpu-request <CPU_REQUEST>] \
+    [--maintenance-job-mem-requestt <MEMORY_REQUEST>] \
```
Comment (ywk253100) [inline site/content/docs/main/velero-install.md:31]: Typo

Comment (qiuming-best) [reply site/content/docs/main/velero-install.md]: updated

File: site/content/docs/main/customize-installation.md (hunk 1 above)
Comment (qiuming-best) [reply site/content/docs/main/customize-installation.md]: updated

**Review by blackpiglet — DISMISSED** [review]

**Review by Lyndon-Li — APPROVED** [review]

**Review by blackpiglet — APPROVED** [review]

**Pull request merged**